MCNPInput.cpp

```diff
@@ -54,6 +54,7 @@ static int makeint( const std::string& token ){
   char* end;
   int ret = strtol(str, &end, 10);
   if( end != str+token.length() ){
+    record << "Warning: string [" << token << "] did not convert to int as expected." << std::endl;
     std::cerr << "Warning: string [" << token << "] did not convert to int as expected." << std::endl;
```

Comment:
It was actually decided to remove the cerr outputs entirely; those changes will be in the next PR, but I could make another small commit to this one removing them as well.